doc/releases/changelog-0.33.0.md

```diff
@@ -188,50 +144,146 @@
   output = example_circuit()
 
   # Graph showing state amplitudes
+  plt.style.use("pennylane.drawer.plot")
   plt.bar(range(len(output)), output)
   plt.show()
   ```
+  
+  <img src="_images/cosine_window.png" width=50%/>
 
-  We can show how this operator is built:
+<h4>New device capabilities, integration with Catalyst, and more! ⚗️</h4>
 
-  ```python
-  import pennylane as qml
+* `default.qubit` now uses the new `qml.devices.Device` API and functionality in
+  `qml.devices.qubit`. If you experience any issues with the updated `default.qubit`, please let us
+  know by [posting an issue](https://github.com/PennyLaneAI/pennylane/issues/new/choose). 
+  The old version of the device is still
+  accessible by the short name `default.qubit.legacy`, or directly via `qml.devices.DefaultQubitLegacy`.
+  [(#4594)](https://github.com/PennyLaneAI/pennylane/pull/4594)
+  [(#4436)](https://github.com/PennyLaneAI/pennylane/pull/4436)
+  [(#4620)](https://github.com/PennyLaneAI/pennylane/pull/4620)
+  [(#4632)](https://github.com/PennyLaneAI/pennylane/pull/4632)
 
-  dev = qml.device("default.qubit", wires=5)
+  This changeover has a number of benefits for `default.qubit`, including:
 
-  op = qml.CosineWindow(wires=range(5))
-  
-  @qml.qnode(dev)
-  def circuit():
-      op.decomposition()
-      return qml.state()
```

Comment:
Took this part out of the CosineWindow entry. Should we show its decomposition?

Reply:
I can't get the decomp to show here easily. Let's leave it out for now but another datapoint for better control over decomposition!